add info about entra group name (#542)
ResearchComputing/Documentation: docs/cloud/aws/lca0/getting-started/managing-customer-groups.md

## Non CU Boulder Users
Non CU Boulder user groups are managed through Microsoft Entra groups.

### Your Group Name

> The Entra group name to manage access your AWS account is in the form '\<account-alias\>-CustomerAdmin'.

Reviewer: ```suggestion
The Entra group name to manage access to your AWS account is in the form `\<account-alias\>-CustomerAdmin`.
```

> To find your account alias look in the top right corner of the screen when you are logged into the AWS console.

Reviewer: ```suggestion
To find your account alias view the top right corner of the screen when logged into the AWS console.

```
